src/parsePref.ts

```diff
@@ -113,12 +115,10 @@ function setGitCommittish (committish: string | null) {
   if (committish != null && committish.length >= 7 && committish.slice(0, 7) === 'semver:') {
     return {
       gitCommittish: null,
-      gitRange: decodeURIComponent(committish.slice(7)),
+      gitRange: committish.slice(7),
```

Comment:
decodeURIComponent is not needed for hosted git repo, as hosted-git-info did decode.

Pushed decodeURIComponent up a level, so line 121 will get the missing decode call.